apps/extension/src/config/chains.ts

```diff
@@ -27,7 +27,9 @@ const defaultChain: Chain = {
   chainName: "Namada Testnet",
   stakeCurrency: ibcCurrency,
   bip44: {
-    coinType: 9999,
+    // coinType = testnet (all coins) - Slip-0044
+    // See: https://github.com/satoshilabs/slips/blob/master/slip-0044.md
+    coinType: 1,
```

Comment:
We would need to support multiple coin types per chain here, right?

Reply:
...or this will be "namada" once we register it there, and the currencies the chain supports are listed under `currencies`.

Reply:
We just need a `slip-0044` address for NAM, not for each coin-type per chain, so once that is registered, we can use that for production. `1` here denotes "testnet"/all-tokens. Having `NAM` registered on [slip-0044](https://github.com/satoshilabs/slips/blob/master/slip-0044.md) will help ensure a user's addresses are unique to our network. That said, we should have a "development" and "production" chain config (in a future PR) with the value set accordingly.